.github/workflows/close-stale-prs.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Close stale issues and PRs'
```

Comment:
Copied directly from https://github.com/hashicorp/tutorials/blob/main/.github/workflows/close-stale-issues-and-prs.yml